llorens/day02/Day02Solver.py

```diff
@@ -0,0 +1,45 @@
+import re
+
+
+def clean_sequence(sequence):
+    return re.sub("\t", " ", sequence)
+
+
+def solve_part_1(sequence):
+
+    sequence = clean_sequence(sequence)
+    count = 0
+
+    for line in sequence.splitlines():
```

Comment:
Is this file management? If that's so it should be separated into another method.